docs/framework/deployment/deployment-guide-for-developers.md

```diff
@@ -51,7 +42,7 @@ For download links, see the section [Redistributable Packages](#redistributable-
 
 - Uninstalling .NET Framework 4.5 or one of its point releases also removes pre-existing .NET Framework 4 files. If you want to go back to .NET Framework 4, you must reinstall it and any updates to it. See [Installing the .NET Framework 4](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/5a4x27ek(v=vs.100)).
```

Comment:
I suggest removing the NOTE on line 28 and replacing occurrences of " .NET Framework 4.5 or one of its point releases" with " .NET Framework 4.5 or later versions".

```suggestion
- Uninstalling .NET Framework 4.5 or later versions also removes pre-existing .NET Framework 4 files. If you want to go back to .NET Framework 4, you must reinstall it and any updates to it. See [Installing the .NET Framework 4](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/5a4x27ek(v=vs.100)).
```

Reply:
Thanks for pointing this out. That's really bad language so I'm going to fix it in this PR.